political/uk/mp_letter_templates/call_to_action.md

```diff
@@ -1,13 +1,12 @@
-// intended for all, probably needs a complete rewrite per country but elements will remain the same
+// Intended for all, probably needs a complete rewrite per country but elements will remain the same
 
-The UK is positioning itself as a world leader in the fight against climate change, however all the work we have done so far is being undermined by the continued 
-growth of proof of work based crypto currencies. Tackling this problem would give significant and immediate returns in the fight agaisnt climate change with 
-limited downsides. At the moment, all cryptocurrencies rely on being freely exchangable for world currencies as their penetration into the market for direct payments
-is very limited. This gives a unique opportunity to shut Bitcoin down before it can do any further damage by cutting it off from the global banking network and 
-prohibiting crypto currency exchanges from operating. 
+The UK is positioning itself as a world leader in the fight against climate change. However, all the work we have done so far is being undermined by the continued 
+growth of proof-of-work based digital tokens; which are commonly referred to by the misnomer crypto “currencies”. Tackling this problem would give significant and immediate returns in the fight against climate change with limited downsides. At the moment, all cryptocurrencies rely on being freely exchangeable for fiat currencies as their penetration into the market for direct payments is very limited. This gives a unique opportunity to shut Bitcoin down before it can do any further damage by cutting it off from the global banking network and prohibiting crypto currency exchanges from operating. 
 
-In November, the UK will be hosting the COP 26 summit for climate change. This presents an ideal oportunity for the UK to take a stand against the horrifically 
-wasteful cryptocurrency industry and further demonstrate it's status as a world leader on tackling climate change. As my MP I hope that you can take this position
-foward and work together with your collegues in parliament to achieve this goal.
+// Focusing on proof-of-work tokens makes it seems as if proof-of-stake is any better. Environmentally yes, but they remain greed inducing pyramid Ponzi schemes.
```

Comment:
This is a good point, should perhaps work on some proof of stake specific snippets.